### DISCUSSION TITLE
Convert project into react js

### DISCUSSION THREAD

**Discussion by Vishnu1804K** [Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-#371] [Ideas]
I want to convert into react so that it will easy for the contributors to implement new features into your project please assign me this issue under GSSOC'24

**Comment by Vishnu1804K** [comment]
Hey @Pratik2401  it is my issue you raise your own issue

**Comment by varshith257** [comment]
@Anishkagupta04 Any thoughts for this migration?

**Comment by Anishkagupta004** [comment]
Can do ....what you say @varshith257

**Comment by varshith257** [comment]
I haven't setup the project and will look into codebase and will suggest my opinion
